pkgs/development/python-modules/shapely/default.nix

```diff
@@ -26,6 +27,14 @@ buildPythonPackage rec {
     hash = "sha256-mX9hWbFIQFnsI5ysqlNGf9i1Vk2r4YbNhKwpRGY7C/Y=";
   };
 
+  patches = [
+    # fixes build error with GCC 14
+    (fetchpatch2 {
```

Comment:
Should use `fetchpatch` here as we don’t need renames and `?full_index=1` is not used in the URL.

Reply:
Done.